### DISCUSSION TITLE
Write patched boot.img failed

### DISCUSSION THREAD

**Discussion by LuckyVirgo** [bmax121/APatch#643] [Q&A]
rom download link: https://drive.usercontent.google.com/download?id=1PurmU6GrddpuTthB72iCT2RETsAUPeOs&export=download&confirm=t&uuid=f3fc9e50-b5f2-4c92-b9f4-5a92a60d9a08

[APatch_bugreport_2024-08-31_16_59.tar.gz](https://github.com/user-attachments/files/16824053/APatch_bugreport_2024-08-31_16_59.tar.gz)
![Screenshot_20240831-170053_APatch](https://github.com/user-attachments/assets/34caee7d-1e67-4662-a6d3-1ca20607df9e)
![Screenshot_20240831-170059_APatch](https://github.com/user-attachments/assets/dfdbfa9a-7ba4-4b5f-9e4e-e38db4a8cc2d)

**Comment by marat2509** [comment]
Try latest nightly
